examples/test.rs

```diff
@@ -10,12 +10,13 @@ extern crate dsp;
 extern crate pitch_calc as pitch;
 extern crate portaudio;
 extern crate synth;
-extern crate time_calc as time;
+extern crate time_calc as timec;
```

Comment:
It looks like this crate import isn't used anywhere in the example. Would you mind removing it instead?